langworld_db_data/adders/listed_value_adder.py

```diff
@@ -225,6 +228,45 @@ def _increment_ids_whose_indices_are_equal_or_greater_than_index_to_assign(
 
         return tuple(rows_with_incremented_indices)
 
+    @staticmethod
+    def _increment_value_ids_in_feature_profiles(
+        new_value_id: str,
+        input_dir: Path,
+        output_dir: Path,
+    ):  # How should I connect it with properties of the object? It has input and output feature profiles dir
+        for file in sorted(list(input_dir.glob("*.csv"))):
+            # I see that in the next function is_changed is used, to write only changed files.
+            # As a first iteration, I am trying to do without it.
+            rows = read_dicts_from_csv(file)
+
+            new_value_id_decomposed = new_value_id.split("-")
+            target_feature_id = f"{new_value_id_decomposed[0]}-{new_value_id_decomposed[1]}"
+            new_value_index = new_value_id_decomposed[2]
+            for i, row in enumerate(rows):
```

Comment:
```suggestion
            for row in rows:
```
I don't think `i` is being used below